terraform/environments/core-logging/route53_resolver_logs.tf

```diff
@@ -15,7 +15,7 @@ data "aws_iam_policy_document" "route53-query-logging-policy" {
       "logs:PutLogEvents",
     ]
 
-    resources = ["arn:aws:logs:*:*:log-group:/aws/route53/*"]
+    resources = ["arn:aws:logs:*:*:log-group:route53-query-logging-policy"]
 
     principals {
       identifiers = ["route53.amazonaws.com"]
```

Comment:
Also this _might_ be incorrect, as we're asking to allow the route53resolver to do the logging, so `route53.amazonaws.com` might be the appropriate thing here.

Reply:
I'll amend it and post the update

Reply:
Apologies! I didn't spell it quite right - `route53resolver.amazonaws.com` should be what we're after, and I've amended my original comment to show this